dev-utils/webdriver.js

```diff
@@ -40,6 +40,11 @@ function isChrome() {
   return browserName.indexOf('chrome') !== -1
 }
 
+function isIOSDevice() {
```

Comment:
not used anywhere. Did you intend to use it here? https://github.com/elastic/apm-agent-rum-js/pull/227/files#diff-058a3bfe50baa67d640de5bc3c79f90bR41

Reply:
Good catch, Wanted to skip timeouts for this version but did not work. I ll give another try or remove this completely.

Reply:
Removed since it did not help.